src/components/url-input-popover/index.js

```diff
@@ -46,57 +71,104 @@ const UrlInputPopover = withState( {
 		'ugb--active': props.newTab || props.noFollow,
 	} )
 
-	return (
-		<Popover
-			className={ mainClassName }
-			focusOnMount={ false }
-			position={ props.position }
-		>
-			<PanelBody>
-				<div className="ugb-url-input-popover__input-wrapper">
-					<Dashicon className="ugb-url-input-control__icon" icon="admin-links" />
-					{ props.onChange && ! props.disableSuggestions && // Auto-suggestions for inputting url.
+	const i18nUrlOptions = urlLabels.map( ( { id, title } ) => ( {
+		id,
+		title: __( title, i18n ), // eslint-disable-line no-restricted-syntax
+	} ) )
+
+	if ( ! LinkControl ) {
```

Comment:
I don't think we still need this if statement (and its contents) anymore as a fallback to the old method since it won't be called anymore unless using WP < 5.4. Please remove